### ISSUE TITLE
Compilation error in Centos 8

### ISSUE THREAD

**Issue by emanmoba** [tomaspinho/rtl8821ce#133]
I am compiling the driver on Centos 8 with kernel version 4.18.0-147.5.1.el8_1.x86_64 on an HP laptop.

I get the following compilation error:

```
  CC [M]  /var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/os_dep/linux/mlme_linux.o
/var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/os_dep/linux/os_intfs.c:1479:22: error: initialization of ‘u16 (*)(struct net_device *, struct sk_buff *, struct net_device *, u16 (*)(struct net_device *, st$
  .ndo_select_queue = rtw_select_queue,
                      ^~~~~~~~~~~~~~~~
/var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/os_dep/linux/os_intfs.c:1479:22: note: (near initialization for ‘rtw_netdev_ops.ndo_select_queue’)
In file included from /var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/include/drv_types.h:30,
                 from /var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/os_dep/linux/pci_intf.c:17:
/var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/include/wifi.h:1034: warning: "IEEE80211_MAX_AMPDU_BUF" redefined


```

And when I add `export USER_EXTRA_CFLAGS=-Wno-error=incompatible-pointer-types` to the Makefile I get this error:

```
/var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/os_dep/linux/rtw_android.c: In function ‘rtw_android_priv_cmd’:
/var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/os_dep/linux/rtw_android.c:668:62: error: macro "access_ok" passed 3 arguments, but takes just 2

  if (!access_ok(VERIFY_READ, priv_cmd.buf, priv_cmd.total_len)) {
                                                              ^
/var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/os_dep/linux/rtw_android.c:668:7: error: ‘access_ok’ undeclared (first use in this function)
  if (!access_ok(VERIFY_READ, priv_cmd.buf, priv_cmd.total_len)) {
       ^~~~~~~~~
/var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/os_dep/linux/rtw_android.c:668:7: note: each undeclared identifier is reported only once for each function it appears in
make[2]: *** [scripts/Makefile.build:313: /var/lib/dkms/rtl8821ce/v5.5.2_34066.20190614/build/os_dep/linux/rtw_android.o] Error 1
make[2]: *** Waiting for unfinished jobs....
```

**Comment by tomaspinho** [comment]
Duplicate of https://github.com/tomaspinho/rtl8821ce/issues/97

**Comment by emanmoba** [comment]
> Duplicate of #97

thanks, but I already read #97. By changing os_intfs.c I get the second error in my original post with `macro "access_ok" passed 3 arguments, but takes just 2 `

**Comment by tomaspinho** [comment]
Remove the first argument of `access_ok` (`VERIFY_READ`) in all calls you can find. This looks like yet another backported change to your kernel. At this point, I won't provide more support because the folks at CentOS are not playing well.

**Comment by emanmoba** [comment]
thanks a lot. Modifying os_intfs.c as described in #97 together with removing the first argument of "access_ok" in rtw_android.c did the trick.

**Comment by wolverine-cs** [comment]
it didn't work for me. could you support me, please?
